samples/exposed-spring/src/main/kotlin/service/UserService.kt

```diff
@@ -0,0 +1,61 @@
+@file:Suppress("InvalidPackageDeclaration")
+
+package org.jetbrains.exposed.samples.spring.service
+
+import org.jetbrains.exposed.samples.spring.domain.User
+import org.jetbrains.exposed.samples.spring.domain.UserEntity
+import org.jetbrains.exposed.samples.spring.domain.UserId
+import org.jetbrains.exposed.sql.SqlExpressionBuilder.eq
+import org.jetbrains.exposed.sql.deleteWhere
+import org.jetbrains.exposed.sql.insertAndGetId
+import org.jetbrains.exposed.sql.select
+import org.jetbrains.exposed.sql.update
+import org.springframework.stereotype.Component
+import org.springframework.transaction.annotation.Transactional
+
+@Component
+@Transactional
+class UserService {
+
+    fun findUserById(id: UserId): User? {
```

Comment:
The function names are descriptive enough, but let's add a small CRUD comment above all 4 just to be consistent with how it's done in [the ktor sample](https://github.com/JetBrains/Exposed/blob/main/samples/exposed-ktor/src/main/kotlin/plugins/Databases.kt). So `findUserById()` would have `// Read user` above it, etc.